### DISCUSSION TITLE
Webbased frontend behind apache reverse proxy and subsite a la mysite.de/zigbee2mqtt

### DISCUSSION THREAD

**Discussion by HalliHalloSchatz** [Koenkk/zigbee2mqtt#13048] [General]
### What happened?

Webbased frontend behind apache reverse proxy and subsite a la mysite.de/zigbee2mqtt uses stoff direct from the main-url and not from the subsite.

mysite.de/assets/xyz not used from mysite.de/zigbee2mqtt/assets/xyz

### What did you expect to happen?

used url:-subsite-setting and the resources from mysite.de/zigbee2mqtt/assets/xyz

### Zigbee2MQTT version

1.26

### Adapter firmware version

20210708

### Adapter

Sonoff Zigbee 3.0 USB Dongle - "type":"zS tack3x0"

**Comment by sjorge** [comment]
You'll need to use mod_substitute to get this to work

```
RedirectMatch /zigbee2mqtt /zigbee2mqtt/
ProxyPass /zigbee2mqtt/ http://localhost:8900/
ProxyPassReverse /zigbee2mqtt/ http://localhost:8900/
RewriteCond %{HTTP:Upgrade} websocket [NC]
RewriteCond %{HTTP:Connection} upgrade [NC]
RewriteRule ^/zigbee2mqtt/api$ "ws://localhost:8900/api" [P,L]
<Location /zigbee2mqtt/>
        AddOutputFilterByType INFLATE;SUBSTITUTE;DEFLATE text/html
        AddOutputFilterByType INFLATE;SUBSTITUTE;DEFLATE text/css
        AddOutputFilterByType INFLATE;SUBSTITUTE;DEFLATE application/javascript
        Substitute "s|/assets/|/zigbee2mqtt/assets/|"
</Location>
````

**Comment by HalliHalloSchatz** [comment]
Wonderful, thank you
